ajax/libs/overpass/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "overpass",
+  "version": "3.0.2",
+  "filename": "overpass-webfont/overpass.css",
```

Comment:
There's no minified version of `overpass.css` available in the repo. Am I supposed to manually create one and add it to my PR?